cli/cmd/release_create.go

```diff
@@ -109,7 +109,7 @@ func (r *runners) setKOTSDefaultReleaseParams() error {
 
 	rev, branch, isDirty, err := r.gitSHABranch()
 	if err != nil {
-		return errors.Wrapf(err, "get git properties")
```

Comment:
i think you can keep this wrap, but `git` prefixes in the other errors make them trackable enough